src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -66,7 +65,6 @@ class BaseSidebarScreen extends Component {
                                 onLayout={this.props.onLayout}
                             />
                         </View>
-                        <KeyboardShortcutsModal />
```

Comment:
We moved the KeybaordShoftcutsModal from `BaseSidebarScreen` to `Expensify.js` as a refactor or for any specific reason?

Reply:
Since it'a high level modal that should displayed at any level, I thought the highest would be the best. We need to mount it at least one. We do something similar with UpdateAppModal for example

Reply:
Agreed with the reasoning, even I wasn't sure why it was in `BaseSidebarScreen`. I asked because the Popover changes worked even without moving the KeyboardsModal to Expensify.js.

Reply:
I think this was to prevent shortcut modal when user logged out.
`BaseSidebarScreen` is highest level on auth screens.
`Expensify.js` is highest level on all screens (including unauthorized screens).

Now I can see shortcut modal open on login page. This is not an issue though 🙂 
![image](https://user-images.githubusercontent.com/108292595/220935832-4de88472-76fd-4108-bf16-dca5c9b7f4b6.png)